code/game/objects/items/weapons/shields.dm

```diff
@@ -34,21 +34,21 @@
 
 /obj/item/weapon/shield/riot/examine(mob/user, distance, infix, suffix)
 	. = ..()
-	var/health_status = (obj_integrity * 100) / (max_integrity-integrity_failure)
+	var/health_status = (obj_integrity-integrity_failure) * 100 / (max_integrity-integrity_failure)
 	if(integrity_failure && obj_integrity <= integrity_failure)
-		. += span_notice("It's broken, it won't protect anymore.")
+		. += span_notice("It's completely broken, with gaping holes everywhere!")
```

Comment:
```suggestion
		. += span_notice("It's completely broken and shattered, with gaping holes all over. It won't be shielding from anything any time soon.")
```

Reply:
Why elaborate it this far? I tweaked the lines multiple times alongside bravemole and a few others in #dev-general and one of the complaints was being too verbose. This goes entirely against that.

Reply:
Also each repair of the shield takes 2 fuel + around 3 seconds to do. I could make the repair take longer, however.

Reply:
Making it 10 + 5% would literally debuff it from the current system... and it's already an issue to the point where changing shields constantly is around 3x as efficient. Fixing a shield as an engineer literally takes longer than applying a splint or two even as an untrained marine.

Reply:
Added numbers to PR description.

Reply:
Lowered integrity per repair to 20% and lengthened the repair time.